.github/workflows/ci.yml

```
@@ -112,15 +112,15 @@ jobs:
       - name: make
         run: |
           apt-get update && apt-get install -y build-essential
-          make -j4 SERVER_CFLAGS='-Werror'
+          make -j4 SERVER_CFLAGS='-Werror' USE_FAST_FLOAT=no
 
   build-macos-latest:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
       - name: make
         # Build with additional upcoming features
-        run: make -j3 all-with-unit-tests SERVER_CFLAGS='-Werror' USE_FAST_FLOAT=yes
+        run: make -j3 all-with-unit-tests SERVER_CFLAGS='-Werror'
```

Dummy change to trigger run-extra-tests.

```suggestion
        run: make -j3 all-with-unit-tests SERVER_CFLAGS='-Werror' USE_FAST_FLOAT=yes
```